WordPress/src/main/java/org/wordpress/android/provider/ProviderUtility.kt

```diff
@@ -0,0 +1,18 @@
+package org.wordpress.android.provider
+
+import com.yarolegovich.wellsql.WellSql
+import dagger.Reusable
+import org.wordpress.android.fluxc.model.QuickStartStatusModel
+import org.wordpress.android.fluxc.model.QuickStartTaskModel
+import javax.inject.Inject
+
+@Reusable
+class ProviderUtility @Inject constructor() {
```

Comment:
I agree that currently this only has functions related to QS; in my intention though this class goes a bit in pair with the ResolverUtility and given the possibly temporary nature of both and not being them meant to be used for other scopes in my understanding, I would not move the relevant functions to QuickStartRepository or other existing classes, so that all of them stay in one place in case we want to remove them in the future. Wdyt?

Reply:
Thanks for considering the effort to unify the providers :+1: 

I've adapted to this in the provider helper for user flags here: https://github.com/wordpress-mobile/WordPress-Android/pull/17423/commits/e7a3859aa76976da6dafa9a4089ffec86ab8e5d9 and also added some null-checks, in case the underlying WellSQL java code can result in null returns.